earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl

```diff
@@ -493,7 +438,7 @@ ()
     if (!IsUpToDate(bound_config, *inputvers)) {
         Mutable${name}AssetD self(GetRef());
         Mutable${name}AssetVersionD newver =
-        MakeNewVersion<Mutable${name}AssetVersionD>(self, bound_config);
+            MakeNewVersion<Mutable${name}AssetD, ${name}AssetImplD::Config, Mutable${name}AssetVersionD>(self, bound_config);
```

Comment:
It seems that these calls to `MakeNewVersion` should not have been changed since `MakeNewVersion` was not touched in this ticket. Can we back out these changes (here and the other 3 below in this file).

Reply:
Never mind. I see that the order of the parameters changed.